src/WidenImage.css

```diff
@@ -0,0 +1,61 @@
+
+.wideImageOverride {
+    /* Center images. */
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+@media (orientation: portrait) {
+    .wideImageOverride {
+        width:100% !important;
+        max-width:100% !important;
+        height:auto !important;
```

Comment:
Ah good point. If the transform works without it I'd think removing it would be best.